components/cloud_usage_modal/mixins.scss

```diff
@@ -0,0 +1,40 @@
+@charset 'UTF-8';
+
+@mixin larger-padding {
```

Comment:
unless these mixins are going to be reused all across the application, I don't see the value of declaring this as mixins instead of simple classes.
3/5 we should try to use css mixins only when either will be highly resused  and probably the values of that mixin can be overwritten (like the button-primary one), or you need to keep the code DRY and you can pass arguments to the mixin to achieve this.